firefox-ios/Client/Frontend/Share/CopyLinkActivity.swift

```diff
@@ -1,24 +0,0 @@
-// This Source Code Form is subject to the terms of the Mozilla Public
-// License, v. 2.0. If a copy of the MPL was not distributed with this
-// file, You can obtain one at http://mozilla.org/MPL/2.0/
-
-import Foundation
-
-class CopyLinkActivity: CustomAppActivity {
-    // Copy link is only available for URL that are not files
-    override func canPerform(withActivityItems activityItems: [Any]) -> Bool {
-        return !url.isFileURL
-    }
-
-    override func prepare(withActivityItems activityItems: [Any]) {}
-
-    override func perform() {
-        UIPasteboard.general.string = url.absoluteString
-        TelemetryWrapper.recordEvent(category: .action,
-                                     method: .tap,
-                                     object: .shareSheet,
-                                     value: .shareCopyLink,
-                                     extras: nil)
```

Comment:
Thanks for checking @lmarceau! I think this is fine. I also was clearing out some dead share-related code in this PR: https://github.com/mozilla-mobile/firefox-ios/pull/23592/files

You'll notice I killed the CopyLink code but I missed this 😆  The old ShareManager's implementation didn't use it anywhere... so this is dead telemetry I guess since this custom item is no longer in the share sheet. 🤷‍♀️

Reply:
Ok thank you! I'll remove the telemetry then 👍